examples/SLIVER1.jl

```diff
@@ -0,0 +1,35 @@
+using BayesMultiFit
```

Comment:
This folder does not know about the AdaptiveMeasurement repository. I suggest changing 'THIS IS AN EXAMPLE OF AN ADAPTIVE MEASUREMENT TYPE AS IT WOULD BE ADDED TO THE AdaptiveMeasurements PACKAGE' to 'THIS IS AN EXAMPLE FOR ADDING AN ADAPTIVE MEASUREMENT.'